Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM sociomantictsunami/dlang:xenial-v3 as builder
```

Comment:
Is this "as" a real thing? I don't get this 2 dockerfiles in one, I never see it before! :scream:

Reply:
https://docs.docker.com/engine/userguide/eng-image/multistage-build

It was added last year afair, I had to use PPA to get recent enough docker in ubuntu.

Reply:
Cool!